src/asana/logic.py

```diff
@@ -0,0 +1,23 @@
+from src.github.models import PullRequest
+from src.github.helpers import pull_request_has_label
+from enum import Enum, unique
+import os
+
+
+@unique
+class AutocompleteLabel(Enum):
+    COMPLETE_ON_MERGE = "complete tasks on merge"
+
+
+def should_autocomplete_tasks_on_merge(pull_request: PullRequest):
+    return (
+        _is_autocomplete_feature_enabled()
+        and pull_request.merged()
+        and pull_request_has_label(
+            pull_request, AutocompleteLabel.COMPLETE_ON_MERGE.value
+        )
```

Comment:
Is there any need to include a check in here to make sure the task isn't already completed? I'd imagine it wouldn't matter if we update it to be completed and it already is.

Reply:
Maybe check whether the API creates another story/notification if you try to complete a task that is already completed?

Reply:
Just tested it and it doesn't generate another story — seems like a no-op